src/components/Target/TargetForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component, PropTypes } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import FormField from '../common/FormField';
+import { validate as validateForm } from 'validate.js';
+import * as constants from '../../constants';
+import '../../styles/Common.scss';
+import Select from 'react-select';
+
+class TargetForm extends Component {
+  constructor(){
+    super();
+    this.state = { value: "two" };
+    this.toggle = this.toggle.bind(this);
+  }
+  toggle(val){
```

Comment:
You can use object shorthand and object matching of javascript ES6 to clean this function.

```
toggle({ value }) {
  this.setState({ value })
}
```